File: app/src/main/java/social/entourage/android/api/model/LoaderCardItem.java
package social.entourage.android.api.model;

import androidx.annotation.NonNull;

import java.util.Date;

public class LoaderCardItem extends TimestampedObject {
    @Override
    public Date getTimestamp() {
        return null;
    }

    @NonNull
    @Override
    public String hashString() {
        return "LoaderCardItem";
    }

    @Override
    public int getType() {
        return LOADER_CARD;
    }

    @Override
    public long getId() {
        return 0;
    }
}

File: app/src/main/java/social/entourage/android/api/model/LoaderCardItem.kt
package social.entourage.android.api.model

import java.util.*

class LoaderCardItem : TimestampedObject() {
    override val timestamp: Date?
        get() = null

    override fun hashString(): String {
        return "LoaderCardItem"
    }

    override val type: Int
        get() = LOADER_CARD

    override val id: Long
        get() = 0
}